beans/src/main/resources/org/ldaptive/beans/spring/spring-ext.xsd

```diff
@@ -109,6 +109,7 @@
       <xsd:extension base="baseAuthenticatorType">
         <xsd:attribute name="baseDn" type="xsd:string" use="required"/>
         <xsd:attribute name="userFilter" type="xsd:string" use="required"/>
+        <xsd:attribute name="passwordAttribute" type="xsd:string" use="optional"/>
```

Comment:
That's my point. This custom schema doesn't currently support the CompareAuthenticationHandler. Support can be added, but it will require more than just this change.

So if you have a use case for using the <ldaptive/> schema, it can be added.
If you're content with using standard spring <bean/> schema, the changes in the beans package can be removed.